hack/for-go-proj.sh

```diff
@@ -19,7 +19,7 @@ set -o pipefail
 set -o nounset
 
 CONTRIB_ROOT="$(dirname ${BASH_SOURCE})/.."
-PROJECT_NAMES=(addon-resizer cluster-autoscaler vertical-pod-autoscaler)
+PROJECT_NAMES=(addon-resizer vertical-pod-autoscaler)
```

Comment:
Since you're removing `cluster-autoscaler` from here, you can remove the condition in line 46.

Reply:
fixed.